### PR TITLE
Pubsub: Test the object form of listenTo

### PR DESCRIPTION
**Pull request by mrjoelkemp**
This is a really helpful feature that isn't documented.

### CODE REVIEW

File: test/specs/trait/pubsub.js [hunk 1]
```diff
@@ -313,6 +313,23 @@ define(['real/trait/pubsub', 'nbd/util/extend'], function(pubsub, extend) {
         obj.trigger('event');
         expect(spy).toHaveBeenCalled();
       });
+
+      it("supports an object of bindings", function() {
```
Comment (mikesherov) [inline test/specs/trait/pubsub.js]: single quotes bro.

**Comment by mikesherov** [comment]
:+1:

File: test/specs/trait/pubsub.js (hunk 1 above)
Comment (mrjoelkemp) [inline test/specs/trait/pubsub.js]: The other specs have double quotes, so I followed suit.

**Comment by Aintaer** [comment]
Just to note, this will release with the next nbd release, which will be along the 1.1 line.

**Pull request merged**